src/rdo.rs

```diff
@@ -475,16 +480,33 @@ fn compute_mean_importance<T: Pixel>(
   total_importance / (bsize.width_mi() * bsize.height_mi()) as f32
 }
 
-fn compute_distortion_bias<T: Pixel>(
+fn compute_activity<T: Pixel>(
   fi: &FrameInvariants<T>, frame_bo: PlaneBlockOffset, bsize: BlockSize,
+  ) -> Option<f64> {
+
+  if bsize == BlockSize::BLOCK_8X8 {
+    fi.activity_mask.mean_activity_of(Rect{x: frame_bo.0.x as isize, y: frame_bo.0.y as isize, width: 8, height:8})
+  }
+  else { None }
+}
+
+fn compute_distortion_bias<T: Pixel>(
+  fi: &FrameInvariants<T>, frame_bo: PlaneBlockOffset, bsize: BlockSize, pli: usize,
 ) -> f64 {
   let mean_importance = compute_mean_importance(fi, frame_bo, bsize);
 
   // Chosen based on a series of AWCY runs.
   const FACTOR: f32 = 3.;
   const ADDEND: f64 = 0.65;
 
-  let bias = (mean_importance / FACTOR) as f64 + ADDEND;
+  let mut bias = (mean_importance / FACTOR) as f64 + ADDEND;
+
+  if pli == 0 {
+    const ACTIVITY_SCALE: f64 = 0.5;
+    let block_activity = compute_activity(fi, frame_bo, bsize);
+    bias /= ACTIVITY_SCALE * block_activity.unwrap_or(1.0_f64 / ACTIVITY_SCALE);
```

Comment:
So, do you intend to apply activity scaler on top of block importance for temporal rdo?
Eventually that is the scenario how rav1e encoder will run, but in the beginning can you consider separate activity masking experiment from temporal rdo? I.e. disable temporal rdo, and your activity scalar is directly applied to the distortion. Otherwise, any possible inteferences btw those two scalars can mislead your experiment?
Also, is this PR still effective? I know that the 1st commit, adding activity masking, is now in separate PR.